diff --git a/src/app/main.cpp b/src/app/main.cpp
index 765bfe3..8d2bd33 100644
--- a/src/app/main.cpp
+++ b/src/app/main.cpp
@@ -233,7 +233,7 @@ int main(int argc, char *argv[])
     }
 
     // Set application-wide window icon. All message boxes and other windows will use it by default.
-    qapp->setDesktopFileName(QStringLiteral("org.zealdocs.zeal"));
+    qapp->setDesktopFileName(QStringLiteral("org.zealdocs.Zeal"));
     qapp->setWindowIcon(QIcon::fromTheme(QStringLiteral("zeal"),
                                          QIcon(QStringLiteral(":/zeal.ico"))));
 
